artifact/local/fs_test.go

```diff
@@ -9,17 +9,19 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/config"
 	"github.com/aquasecurity/fanal/cache"
 	"github.com/aquasecurity/fanal/types"
 )
 
-func TestArtifact_Inspect(t *testing.T) {
+func Test_localArtifact_Inspect(t *testing.T) {
```

Comment:
same as above, changed to make it easier to trace hashes change, I forgot to change it back :facepalm: